qtodotxt/lib/task_parser.py

```diff
@@ -1,15 +1,29 @@
 import re
+import string
 from datetime import datetime, date
+from qtodotxt.lib import settings
 
 
-HIGHER_PRIORITY = 'A'
-LOWER_PRIORITY = 'Z'
+HIGHEST_PRIORITY = 'A'
+LOWEST_PRIORITY = 'Z'
```

Comment:
and do not use uppercae global variable, just QSettings, then the settings will be available under .config/QTodoTxt/QTodoTxt.conf with a simple INI format